client/src/Movies/Movie.js

```diff
@@ -11,7 +11,7 @@ export default class Movie extends Component {
 
   componentDidMount() {
     // change this line to grab the id passed on the URL
-    const id = 1;
+    const id = this.props.match.params.id;
```

Comment:
great job, you could also destructure it like this...  const { id }  = this.props.match.params